src/Calypso-SystemTools-Core/ClyRequestMethodTagForEditorCommand.class.st

```diff
@@ -25,3 +29,13 @@ ClyRequestMethodTagForEditorCommand >> execute [
 	tagEditor := methodEditor tagsAndPackageEditor.
 	tagEditor requestChangeBy: [ tagEditor requestTag] 
 ]
+
+{ #category : #'as yet unclassified' }
+ClyRequestMethodTagForEditorCommand >> readParametersFromContext: aToolContext [
+
+	menuItemName := aToolContext tool hasUnacceptedEdits 
```

Comment:
Idea was to refactor superclass  ClyMethodEditorCommand prepare method: 
```Smalltalk
ClyMethodEditorCommand>>prepareFullExecutionInContext: aToolContext
	super prepareFullExecutionInContext: aToolContext.
	
	methodEditor := aToolContext tool
```
It should be replaced by 
```Smalltalk
ClyMethodEditorCommand>>readParametersFromContext: aToolContext
	super readParametersFromContext: aToolContext.
	
	methodEditor := aToolContext tool
```
And then methodEditor can be used to implement conditional #defaultMenuItemName:
```Smalltalk
ClyRequestMethodTagForEditorCommand>>defaultMenuItemName
    methodEditor hasUnacceptedEdits
                ifTrue: [ 'Classify currently editing method' ]
 		ifFalse: [ 'Classify method' ]
```